HIP/hip-991.md

```diff
@@ -132,20 +132,21 @@ The HIP parameters are defined as follow:
 ### User Flows and Interaction
 
 * Users will specify the fee settings during the topic creation process through a simple interface in their Hedera client (refer to the creation of token custom fees/fixed fee for reference).
-* When submitting a message to a topic with custom fees through an application or wallet interface, users must set the maximum fee for the message. Alternatively, users can add a flag to accept all custom fees from a topic id. If the topic has no custom fees, neither is required.
+* When submitting a message to a topic with custom fees through an application or wallet interface, users can set the maximum fee for the message. If `max_custom_fee` is not set, any custom fee will be accepted by the user (up to the payer's available funds).
 * In case of user wallets, applications show the custom fees to the user before submitting the message.
 * Operators will get fee collections and distributions automatically through the custom fees just like they do in the token service currently.
+* If the specified custom fee limit or the payer's funds are insufficient to cover the custom fees, the transaction will fail.
```

Comment:
should we also highlight that when max custom fee is specified and any custom fee the topic intends to charge without a matching max custom fee will also fail the transaction?

Reply:
Yes, makes sense.